decidim-proposals/spec/events/decidim/proposals/publish_proposal_event_spec.rb

```diff
@@ -59,6 +59,17 @@ module Proposals
         end
       end
 
+      context "when the proposal is official" do
+        let(:resource) { create :proposal, :official, title: "A nice proposal" }
```

Comment:
I think you need to define the correct extra details for the event in this spec for it to make sense in the given context:
```suggestion
        let(:resource) { create :proposal, :official, title: "A nice proposal" }
        let(:extra) { { participatory_space: resource.participatory_space } }
```

This way it will **actually** test the related issue you are fixing in this PR.